platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -272,6 +272,13 @@ struct ipsec_sa_s {
 			} out;
 		};
 	} sa_info;
+
+	/*
+	 * Flag to check if the SA expiry status event was already
+	 * sent. This is applicable only for IPsec SA in inline outbound
+	 * mode.
+	 */
+	odp_atomic_u32_t status_notify;
```

Comment:
This should be renamed to be more descriptive, e.g. expiry_notified or expiry_msg_sent.

It would also be good to move this before stats in the SA structure as this field may get referenced for every packet unlike sa_info and stats.